_includes/feed.liquid

```diff
@@ -4,13 +4,14 @@ section: smartContract
 ---
 
 {% if metadata.solanafeeds %}
-    <p>The program address for this feed is publicly available in the <a href="https://explorer.solana.com/address/2yqG9bzKHD59MxD9q7ExLvnDhNycB3wkvKXFQSpBoiaE?cluster=devnet">Solana Devnet Explorer</a>.</p>
+    <p>To learn how to implement these feeds, see the <a href="../get-the-latest-price/#solana-network-examples">Solana Examples for Consuming Price Feeds</a>.</p>
```

Comment:
Let's keep the same wording for all of our feeds.
`To learn how to implement these feeds, see..` -> `For implementation instructions, see..`

Reply:
Good call. It's best to be consistent. I still recommend that we re-word the sentence. Does that work, or do you prefer the existing sentence?